Automattic/wp-calypso
# Use abtests from Calypso

#### Changes proposed in this Pull Request

* This changes e2e tests from using default config for abtests to using active-tests.js

#### Testing instructions
* Ensure tests pass in CircleCI
* Run locally an make sure that after `setABTestControlGroupsInLocalStorage` is run, all tests and defaults from active-tests.js are set properly in local storage

Before:
![image](https://user-images.githubusercontent.com/31110506/55001357-080ae380-4f9a-11e9-8fff-96ac18c4677e.png)

After:
![image](https://user-images.githubusercontent.com/31110506/55001138-903cb900-4f99-11e9-8e10-fb40e8daa57b.png)
